config/services/dataplane_v1beta1_openstackdataplaneservice_ddp_package_option.yaml

```diff
@@ -3,4 +3,5 @@ kind: OpenStackDataPlaneService
 metadata:
   name: ddp-package-option
 spec:
+  label: ddp-package-option
```

Comment:
we don't use label anymore https://github.com/openstack-k8s-operators/dataplane-operator/pull/713

Reply:
@fao89 thanks. will close this PR.